python/speech-to-text/README.MD

```diff
@@ -0,0 +1,47 @@
+
+# Speech To Text using Python (google, aws, azure)
+
+### Example Input
+```
+{"provider":"aws","language":"en","audio":"iVBORw0KGgoAAAANSUhEUgAAAaQAAALiCAY...QoH9hbkTPQAAAABJRU5ErkJggg=="}
+```
+
+key
+a dict which provides `key` and `region` of the asked cloud provider in payload like.
+key = 
```

Comment:
What's with the `key` and `key =`?

Reply:
I am just explaining there the format of the Google key but I will import it from the environment rather than passing it as a function argument.